src/02-perturbing-a-two-state-system.md

```diff
@@ -132,7 +132,7 @@ Just as in the previous tutorial (see Figs 3 & 4), we see Rabi oscillations in t
 1. The period of oscillations has gone from 31 to about 14 (recall, we have kept A the same)
 2. Instead of a complete oscillation from 0 to 1 of both states, we see that we are more likely to find the state as |+>. 
 
-We can understand 2 by recalling that the perturbation creates an energy difference between the |+> and |->.  We can think of the perturbation as a barrier between the two states - the larger the barrier the less effective the coupling between the states will be and the more likely you'll stay in the state you started in.
+We can understand 2 by recalling that the perturbation creates an energy difference between the |+> and |->.  We can think of the perturbation as a splitting of the two level eigenvalues (at zero coupling) due to some external interaction. In the case of two wells separated by a barrier, the introduction of the $\delta$ corresponds to a lowering/raising of one of the two wells, keeping the other fixed and without changing the barrier in between.
```

Comment:
I think you are spot on technically. What I was trying to do here is give some intuitive sense of why the oscillations in probability don't go from full 0 to 1 i.e. when we start in the + state we more of less stay there. From what you've written I don't get a sense of why that is. In general, perhaps the problem here is that I have triggered the thought in our minds of two coupled wells and barriers before we are quite ready for them. From the discussion on the [issue you raised](https://github.com/project-ida/two-state-quantum-systems/issues/9) my barrier explanation is just plain wrong so I think we should stay away from it for now.

Could we instead make reference to coupled pendulums like we did in the first tutorial? In particular I found [this video](https://www.youtube.com/watch?v=Z5rKTagEsro) (unfortunately not a real life video) of coupled pendulums of different lengths (i.e. different frequencies). There you can see that the ones with different lengths do oscillate but just not as much.

Then we could say something like

```suggestion
We can understand 2 by recalling that the perturbation creates an energy difference between the |+> and |->. This gives rise to a frequency difference much like what we'd have in the classical coupled pendulum problem if the lengths of the pendulums were different. In that case the coupling is much less effective and most of the energy stays in pendulums of the same length as can be seen in this [simulation video](https://www.youtube.com/watch?v=Z5rKTagEsro)
```

Reply:
I just committed the change